### DISCUSSION TITLE
Yarr on legitimate copy with 4gb patch

### DISCUSSION THREAD

**Discussion by khirasier** [tsunamods-codes/7th-Heaven#224] [General]
7th heaven: version 4.0

if you use the 4gb patch on the exe then 7th heaven will detect the game as pirated and refuse to launch the game.

**Comment by julianxhokaxhiu** [comment]
On which exe did you apply it? We do recognize the 4GB patch on those just fine.

**Reply by emcatalan** [reply]
I also have the same Issue.  I have the steam version I patched the ff7_en.exe using the ntcore 4gb patch.  After patching, 7th heaven detects it is pirated but it still runs on steam with ffnx without problems.
